pagopa/io-app
# chore: [IOPAE-1050] Add service preferences to `ServiceDetailsScreen`

This PR depends on https://github.com/pagopa/io-app/pull/5642

## Short description
This PR adds the preferences section in the service details screen.

<details open><summary>Details</summary>
<p>

| preferences  |
| - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/709c75b9-0728-49bf-b942-01c9f0937a57" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- moved service reducers to the _/features/services/store/reducers/_ folder
- moved service sagas to the _/features/services/saga/_ folder
- moved service actions to the _/features/services/store/actions/_ folder
- added ServiceDetailsPreferences component
- added ServicePreferenceListItemSwitch component
- updated ServiceDetailsScreen in order to display service preferences via ServiceDetailsPreferences

## How to test
Using `io-dev-api-server`, navigate to the services tab and tap a service. Check that the screen is displayed correctly.
_Note: the inbox switch is disabled if the selected service is special._
